scripts/examples/python/setup.cfg

```diff
@@ -0,0 +1,27 @@
+[metadata]
+name = XenAPI
+version = attr: src.VERSION
+description = XenAPI SDK, for communication with Citrix Hypervisor.
+author = Citrix Systems, Inc.
+author_email = xen-api@lists.xenproject.org
+url = http://community.citrix.com/display/xs/Download+SDKs
```

Comment:
That has a lot of 404s, would this be a better starting point: https://developer-docs.citrix.com/citrix-hypervisor/

Reply:
I think this might be better https://www.citrix.com/community/citrix-developer/citrix-hypervisor-developer/

Reply:
It is a bit difficult to find it from there:
https://www.citrix.com/community/citrix-developer/citrix-hypervisor-developer/citrix-hypervisor-developing.html 404
https://developer-docs.citrix.com/ works but have to choose CH again to get to:
https://developer-docs.citrix.com/citrix-hypervisor/

How about we use https://developer-docs.citrix.com/citrix-hypervisor/ as the link, or make https://www.citrix.com/community/citrix-developer/citrix-hypervisor-developer/citrix-hypervisor-developing.html  redirect to it so its easier to find?

Reply:
I don't think it is correct to link to the docs page, it is too restricted. We should be linking to the main developer page for citrix hypervisor. The 404s have to be raised as bugs with the relevant maintaining team and fixed, but they are not the python package's problem.

Reply:
Docs in the SDK about python don't match the usecase of using pypi to install the package: https://developer-docs.citrix.com/projects/citrix-hypervisor-sdk/en/latest/getting-started/#python

I'd be more inclined to use developer-docs.citrix.com/citrix-hypervisor as a homepage, even if it's just because it's easier to go to the docs about python.

In the end it might be better to just direct users to this repository as it's more practical for them.